action.yml

```diff
@@ -11,6 +11,11 @@ inputs:
     description: Push the changes to a branch instead of publishing to the NPM registry.
   custom_message:
     description: Custom message to added to the beginning of the release GitHub comment.
+  install_instructions:
+    description: How to show the install instructions of the published packages. Use 'global' to show the install command for global packages. Defaults to a local dependencies json example.
```

Comment:
I think this could be replaced or simplified with a flag called `is_global`. I can't imagine any other reasons for needing this right now.